### ISSUE TITLE
Module compiled with Swift 4.1 cannot be imported by the Swift 4.2 compiler

### ISSUE THREAD

**Issue by olcayertas** [regulaforensics/DocumentReader-iOS#6]
Please recompile your framework with Xcode 10 and republish it on Cocoapods.

**Comment by manhlx3006** [comment]
You could try to add this code to end of your Podfile as well.

post_install do |installer|
  installer.pods_project.targets.each do |target|
    if ['DocumentReader'].include? target.name
      target.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = 'change_to_your_compatible_compiler_version'
      end
    end
  end
end

**Comment by olcayertas** [comment]
@DmitrySmolyakov shoul i also add `beta` to `DocumentReaderFull`?

**Comment by DmitrySmolyakov** [comment]
@olcayertas No, DocumentReaderCore.framework (pod DocumentReaderFull) was compiled as Objective C framework, so it worked fine with any versions of xCode
